app/(game)/(components)/ClearedBlock.tsx

```diff
@@ -0,0 +1,142 @@
+import { Sprite, useTick } from "@pixi/react";
+import { useMemo, useState, useRef } from "react";
+
+import {
+  GAME_PANEL,
+  TETROMINO_STYLES,
+  TetrominoType,
+  randomFloatInRange,
+} from "./Utils";
+
+type ClearedBlockProps = {
+  type: TetrominoType;
+  initialXDisplacement: number;
+  initialYDisplacement: number;
+};
+
+type SpriteProperties = {
+  alpha: number;
+  position: [number, number];
+  scale: number;
+};
+
+const ANIMATION_SPEED_MULTIPLIER = 5;
+
+/* Max animation duration in seconds */
+const MAX_ANIMATION_DURATION = 3;
```

Comment:
nitpick here but we should add units on constant's name (i.e. `MAX_ANIMATION_DURATION_S`)